docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Concatenate.cs

```diff
@@ -1,65 +1,73 @@
 ﻿using System;
+using System.Collections.Generic;
 using Microsoft.ML;
 using Microsoft.ML.Data;
 
 namespace Samples.Dynamic
 {
-    public static class ConcatTransform
+    public static class Concatenate
     {
         public static void Example()
         {
             // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
             // as well as the source of randomness.
             var mlContext = new MLContext();
 
-            // Get a small dataset as an IEnumerable and them read it as ML.NET's data type.
-            var data = Microsoft.ML.SamplesUtils.DatasetUtils.GetInfertData();
-            var trainData = mlContext.Data.LoadFromEnumerable(data);
+            // Create a small dataset as an IEnumerable.
+            var samples = new List<InputData>()
+            {
+                new InputData(){ Feature1 = 0.1f, Feature2 = new[]{ 1.1f, 2.1f, 3.1f}, Feature3 = 1 },
+                new InputData(){ Feature1 = 0.2f, Feature2 = new[]{ 1.2f, 2.2f, 3.2f}, Feature3 = 2 },
+                new InputData(){ Feature1 = 0.3f, Feature2 = new[]{ 1.3f, 2.3f, 3.3f}, Feature3 = 3 },
+                new InputData(){ Feature1 = 0.4f, Feature2 = new[]{ 1.4f, 2.4f, 3.4f}, Feature3 = 4 },
+                new InputData(){ Feature1 = 0.5f, Feature2 = new[]{ 1.5f, 2.5f, 3.5f}, Feature3 = 5 },
+                new InputData(){ Feature1 = 0.6f, Feature2 = new[]{ 1.6f, 2.6f, 3.6f}, Feature3 = 6 },
+            };
 
-            // Preview of the data.
-            //
-            // Age    Case  Education  induced     parity  pooled.stratum  row_num  ...
-            // 26.0   1.0   0-5yrs      1.0         6.0       3.0      1.0  ...
-            // 42.0   1.0   0-5yrs      1.0         1.0       1.0      2.0  ...
-            // 39.0   1.0   0-5yrs      2.0         6.0       4.0      3.0  ...
-            // 34.0   1.0   0-5yrs      2.0         4.0       2.0      4.0  ...
-            // 35.0   1.0   6-11yrs     1.0         3.0       32.0     5.0  ...
+            // Convert training data to IDataView.
+            var dataview = mlContext.Data.LoadFromEnumerable(samples);
 
-            // A pipeline for concatenating the Age, Parity and Induced columns together into a vector that will be the Features column.
+            // A pipeline for concatenating the "Feature1", "Feature2" and "Feature3" columns together into a vector that will be the Features column.
             // Concatenation is necessary because learners take **feature vectors** as inputs.
```

Comment:
>** [](start = 64, length = 2)

remove the **. 

learners take vectors of floats

Reply:
also should trainers be used instead of learners?

---
In reply to: [273801982](https://github.com/dotnet/machinelearning/pull/3262#discussion_r273801982) [](ancestors = 273801982)